aqua/iTerm2/Portfile

```diff
@@ -55,6 +55,7 @@ github.livecheck.regex {(\d+(?:\.\d+)*)}
 
 post-patch {
     reinplace "s|CODE_SIGN_IDENTITY = \".*\";|CODE_SIGN_IDENTITY = \"\";|g" ${worksrcpath}/iTerm2.xcodeproj/project.pbxproj
+    reinplace "s|enableUBSanitizer = \"YES\"||g" ${worksrcpath}/iTerm2.xcodeproj/xcshareddata/xcschemes/iTerm2.xcscheme
```

Comment:
Using the Development build target is the root cause of this problem. Any reason you don't use Release instead? It will be quite a bit faster. Development tends to change and you'll likely have similar problems in the future (e.g., if ASAN gets enabled).

Reply:
Let me give that a try - I'm not sure I knew that was the problem. I've been trying to move this to the XCode portgroup with not a lot of luck - but that could be a quick fix.

Reply:
The build runs `make prod` which I believe is using Release target? Through my experiment I've found it's the `-derivedDataPath ./DerivedData` arg added to fix permission errors causing the problem, even if you move this to Xcode pg this will still be needed, and will still trigger the link to UBSan.

Reply:
[Portfile.zip](https://github.com/macports/macports-ports/files/7724488/Portfile.zip)
This is my attempt to move to Xcode pg, in case you're interested.

Reply:
Yeah, `make prod` should be unaffected by that change. I don't see where in the Portfile it calls make, though. Do you have the output from `make`? That should make it very clear what build target is actually being used.

Reply:
If no other portgroups (cmake/xcode/meson etc.) is used then `./configure && make` is the default and we have configure script disabled in this port. Full build log is here https://build.macports.org/builders/ports-12_x86_64-builder/builds/5819/steps/install-port/logs/stdio